vizro-core/src/vizro/static/css/token_names.css

```diff
@@ -14,11 +18,11 @@
   --status-light-mode-error: rgba(240, 59, 58, 1);
   --status-light-mode-information: rgba(0, 158, 255, 1);
   --surfaces-dark-mode-bg-01: rgba(35, 38, 50, 1);
-  --surfaces-dark-mode-bg-02: rgba(27, 30, 42, 1);
+  --surfaces-dark-mode-bg-02: #1b1e2a;
   --surfaces-dark-mode-bg-03: rgba(20, 23, 33, 1);
   --surfaces-dark-mode-bg-card: rgba(35, 38, 50, 1);
   --surfaces-light-mode-bg-01: rgba(255, 255, 255, 1);
-  --surfaces-light-mode-bg-02: rgba(245, 246, 246, 1);
+  --surfaces-light-mode-bg-02: #f5f6f6;
```

Comment:
Can we actually revert the changes here? The rgbas are the existing hex colors and we're using rgba throughout the entire file, I would just stick with it to be consistent